UnitsNet/Scripts/UnitDefinitions/Length.json

```diff
@@ -70,6 +70,18 @@
       ]
     },
     {
+      "SingularName": "USSurveyFoot",
+      "PluralName": "USSurveyFeet",
+      "FromUnitToBaseFunc": "x*0.3048006096",
+      "FromBaseToUnitFunc": "x/0.3048006096",
```

Comment:
As per your comment
> Equal to exactly 1200/3937 meters

and also swapping the functions (see my previous comment about your test value being wrong), I would change these conversions to:
```
"FromUnitToBaseFunc": "x*3937/1200",
"FromBaseToUnitFunc": "x*1200/3937",
```

Now this reads as expected; to convert one meter (base unit) to USS feet, multiply by 1200 and divide by 3937, a factor of about 0.3.

This way we use the most accurate definition instead of an artificially limited factor. If we later go from double to decimal, it would automatically get a better conversion precision.